### DISCUSSION TITLE
ValueError:  [!] Model file not found in the output path

### DISCUSSION THREAD

**Discussion by mehransalehi** [coqui-ai/TTS#2201] [General Q&A] [answered]
Hello 
This is my first time to use coqui .
I installed it by following the documentation but when i run test command it gave me this output : 

`
$ tts --text "Text for TTS" --out_path ./speech.wav                                                                                                                                                 
 
 > tts_models/en/ljspeech/tacotron2-DDC is already downloaded.
 > vocoder_models/en/ljspeech/hifigan_v2 is already downloaded.
Traceback (most recent call last):
  File "/home/phoenix/.local/bin/tts", line 8, in <module>
    sys.exit(main())
  File "/home/phoenix/.local/lib/python3.7/site-packages/TTS/bin/synthesize.py", line 298, in main
    vocoder_path, vocoder_config_path, _ = manager.download_model(args.vocoder_name)
  File "/home/phoenix/.local/lib/python3.7/site-packages/TTS/utils/manage.py", line 245, in download_model
    output_model_path, output_config_path = self._find_files(output_path)
  File "/home/phoenix/.local/lib/python3.7/site-packages/TTS/utils/manage.py", line 268, in _find_files
    raise ValueError(" [!] Model file not found in the output path")
ValueError:  [!] Model file not found in the output path
`

**Comment by mehransalehi** [comment, marked as answer]
Ok i solve it .
in output it mentioned that vendor is downloaded but it's not then i delete the content of 
`
/home/user/.local/share/tts/
`
it download it again and problem solved.

**Reply by TaterTotX** [reply]
Hello, I also encountered the same problem. May I ask how did you solve it?



from TTS.api import TTS

model_name = TTS.list_models()[0]
tts = TTS(model_name)
wav = tts.tts("This is a test! This is also a test!!", speaker=tts.speakers[0], language=tts.languages[0])


raise ValueError(" [!] Model file not found in the output path")
ValueError:  [!] Model file not found in the output path

**Reply by erogol** [reply]
Please do not triple post

**Reply by sangthebui** [reply]
Thank you for finding this out.

For other people who uses Mac, ~/Library/Application Support/tts is their path. I had to delete this path, not the /home/user/.local/share/tts/ path.

**Reply by mikee-gwu** [reply]
Thank you for the MacOS pointer, just what I needed. Its too bad the cli doesn't have some sort of --force_download or something to help with partial/failed downloaded models, but this helps fix. thank you!

**Comment by tonywhite11** [comment]
2 days, 3 nights and chat GPT4 and I still couldn't figure it out. Can't thank you enough.

**Comment by Manamama** [comment]
I have spent some time figuring this out, too. 
In short: 
`rm -rf  ~/.local/share/tts`
should do the trick in Ubuntu. 
Still the program needs to provide for such a check and an option do remove the broken download.

**Reply by SuperGoodGame** [reply]
I dont know how to reinstall it in linux ，can you tell me how to delete the error package and reinstall it？

**Comment by jp-x-g** [comment]
It goes without saying based on the comments that have been made here, but for reference, the actual error has nothing to do with the error message, or with a "model file not found in the output path" -- it happens when the model file is incomplete or corrupt for some reason (i.e. a download that was interrupted), and can be fixed by removing or renaming the folder for that model and re-downloading it.
